README.md

```diff
@@ -114,14 +115,15 @@ If you are looking for the codebase of the current production version of GATK, p
     * Running a Spark tool on a cluster requires Spark to have been installed from http://spark.apache.org/, since
       `gatk-launch` invokes the `spark-submit` tool behind-the-scenes.
 
-* To run a Spark tool on Google Cloud Dataproc:
+* To run a GATK Spark tool on Google Cloud Dataproc:
     * You must have a [Google cloud services](https://cloud.google.com/) account, and have spun up a Dataproc cluster
-      in the [Google Developer's console](https://console.developers.google.com).
+      in the [Google Developer's console](https://console.developers.google.com). You may need to have the "Allow API access to all Google Cloud services in the same project" option enabled (settable when you create a cluster).
     * You need to have installed the Google Cloud SDK from https://cloud.google.com/sdk/, since
       `gatk-launch` invokes the `gcloud` tool behind-the-scenes. As part of the installation, be sure
       that you follow the `gcloud` setup instructions [here](https://cloud.google.com/sdk/gcloud/).
     * Your inputs to the GATK need to be in Google Cloud Storage buckets, and should be specified on
       your GATK command line using the syntax `gs://my-gcs-bucket/path/to/my-file`
+   * You may need to pass your credentials explicitly, e.g., to pass the API key use the `--apiKey` argument to GATK (you can create an API key on the Credentials tab of the API Manager page, i.e., `https://console.cloud.google.com/apis/credentials?project=<your_gcloud_project>`
```

Comment:
Also, this URI is likely to get stale in the future -- recommend saying just "you can create an API key on the Credentials tab of the API Manager page"